### DISCUSSION TITLE
GitHub Actions? What’s the correct way to do this in a YML file?

### DISCUSSION THREAD

**Discussion by swchips** [community/community#136912] [Actions] [answered]
### Select Topic Area

Question

### Body

How do I set and use environment variables in GitHub Actions? What’s the correct way to do this in a YML file?

**Comment by Haruniv** [comment, marked as answer]
Hi, chips

You can define and use environment variables in GitHub Actions using the env keyword. Here’s an example!

e.g. yaml file

jobs:
  build:
    runs-on: ubuntu-latest
    env:
      NODE_ENV: production
    steps:
      - name: Echo Environment Variable
        run: echo $NODE_ENV


In this example, the NODE_ENV environment variable is set to production and is then used in the echo $NODE_ENV command.
